core/internal/runfiles/uploader.go

```diff
@@ -244,14 +244,14 @@ func (u *uploader) scheduleUploadTask(
 ) {
 	localPath := filepath.Join(u.settings.GetFilesDir(), relativePath)
 	task := &filetransfer.Task{
+		// TODO: Set FileKind
```

Comment:
is this TODO needs to be addressed?

Reply:
in a separate PR, since the changes might be somewhat large but localized to `uploader.go`

without this, all files uploaded via this codepath are counted as "other" rather than "wandb" / "artifact" / "media"

Reply:
oh got ya! sounds good!